openedx_tagging/core/tagging/models/base.py

```diff
@@ -239,12 +260,11 @@ def copy(self, taxonomy: "Taxonomy") -> "Taxonomy":
         self.required = taxonomy.required
         self.allow_multiple = taxonomy.allow_multiple
         self.allow_free_text = taxonomy.allow_free_text
-        self.system_defined = taxonomy.system_defined
         self.visible_to_authors = taxonomy.visible_to_authors
         self._taxonomy_class = taxonomy._taxonomy_class
         return self
 
-    def get_tags(self) -> List[Tag]:
+    def get_tags(self, tag_set: models.QuerySet = None) -> List[Tag]:
```

Comment:
I forgot to document this added parameter here -- @ChrisChV could you add `tag_set` to the docstring?